src/Header.tsx

```diff
@@ -186,3 +222,16 @@ function MobileHeader({ anchor, currentTab }) {
     <Toolbar id={anchor} />
   </>
 }
+
+function isDbVersionOlder(dbVersion: string) {
```

Comment:
just store the patch as a string, and compare current patch string vs stored string. no need to amortize into a number, when will we never have a decrease in patch number?

Reply:
When we change minor or major, then patch will reset to 0. If the stored version was 8.9.1, and the new version is 8.10.0, then comparing just patch will not mark the patch notes as new.

Also, if someone last used GO on 8.9.2, then comes back on 8.10.1, it will also not be marked as new.

If I just compare the whole string, then the move from something like 8.9.0 to 8.10.0 will not be marked as new.
<img width="114" alt="image" src="https://user-images.githubusercontent.com/36019388/195173207-59b4dfa2-7060-4409-84f5-5410375262c5.png">

Reply:
why are you checking if a version number is higher or not?
![image](https://user-images.githubusercontent.com/1754901/195183361-aa925a92-8438-4ba8-b835-9c773767f294.png)

Reply:
Also, storing the data in localstorage is "correct", but it might get destroyed by the database bundling, causing weird behaviour. 
https://github.com/frzyc/genshin-optimizer/pull/665 is suppose to fix that.